Wrapping/Python/Examples/use_case_02.py

```diff
@@ -36,6 +45,35 @@ def import_csv_xyz_vertices_2(file_path: str):
     dca.addOrReplaceDataContainer(dc)
     # Write the Vertex Geometry out to a .dream3d file.
     err = simpl_helpers.WriteDREAM3DFile('vert_geom_2.dream3d', dca, True)
```

Comment:
This line needs to be moved to the bottom of the function so that everything that got added to the DataStructure hierarchy is written to the DREAM3D file.